pkg/storage/tsdb/tombstones.go

```diff
@@ -0,0 +1,321 @@
+package tsdb
```

Comment:
Shouldn't this code be tested?

Reply:
Yes it should. I have added the unit tests for this now as well.